src/components/Layout/components/spacing/spacing.tsx

```diff
@@ -0,0 +1,33 @@
+import {block} from '../../../utils/cn';
+import {Space} from '../../types';
+
+import './spacing.scss';
+
+const b = block('s');
+
+export interface SpacingProps {
```

Comment:
Let's add comment for interface describing that props names are short versions of styles. Ex.`mr` is for `margin-top`, `pt` is for `padding-top`, plus description that `y` and `x` can be used for one direction spacing. It would be useful.